client/setup-enzyme-adapter.js

```diff
@@ -0,0 +1,4 @@
+import { configure } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+
+configure({ adapter: new Adapter() });
```

Comment:
Don't forget to add a newline at the end of all of your files! Do you have a eslint running in your IDE?